### PR TITLE
don't forget to sign addon repository

### PR DESCRIPTION
**Pull request by wfeldt**

### CODE REVIEW

File: mksusecd [hunk 1]
```diff
@@ -1775,6 +1775,18 @@ sub run_createrepo
...
 
   die "error: createrepo failed\n" if !$ok;
+
+  # sign repomd.xml
+
+  my $name = "$dir/repodata/repomd.xml";
+
+  return if !$sign_key_dir || !-f $name;
+
+  system "cp $sign_key_pub $name.key";
```
Comment (jreidinger) [inline mksusecd]: It cannot have any strange character inside? I think that at least using `''` can help`

Comment (wfeldt) [reply mksusecd]: No, it's a temp file name.

Comment (jreidinger) [reply mksusecd]: e.g. if `$dir` contain space

**Comment by sriemer** [comment]
Reported by me. I can confirm this is fixed. Thanks!

**Pull request merged**